test/run/init_app.js

```diff
@@ -0,0 +1,20 @@
+const { spawn } = require( 'child_process' );
+const { BUILT_APP_DIR } = require( './paths' );
+
+function initApp( log ) {
+    const startCmd = './'
```

Comment:
was this more significant before? what's the purpose of splitting it from `./WordPress.com`?

Reply:
This was in anticipation of Windows-specific syntax, however I've been able to determine that will be unnecessary and we can stick with `./WordPress.com`, which is nice.